wcivf/apps/people/models.py

```diff
@@ -8,6 +8,7 @@
 from django.utils.functional import cached_property
 from elections.models import Election, Post
 from parties.models import Party
+from django.utils.translation import gettext_lazy as _
```

Comment:
These changes introduced some test failures so I don't think I've done this correctly.